yt_dlp/extractor/cbs.py

```diff
@@ -99,25 +156,37 @@ def _extract_video_info(self, content_id, site='cbs', mpx_acc=2198311517):
                     'Downloading %s SMIL data' % asset_type)
             except ExtractorError as e:
                 last_e = e
-                continue
+                if useXMLmetadata:
+                    continue
+                query['formats'] = ''  # blank query to check if expired
+                try:
+                    tp_formats, tp_subtitles = self._extract_theplatform_smil(
+                        update_url_query(tp_release_url, query), content_id,
+                        'Downloading %s SMIL data, trying again with another format' % asset_type)
+                except ExtractorError as e:
+                    last_e = e
+                    continue
             formats.extend(tp_formats)
             subtitles = self._merge_subtitles(subtitles, tp_subtitles)
```

Comment:
Is this correct, or should these formats be ignored?

Reply:
not sure what you're asking here sorry. maybe this answers your question.? https://github.com/yt-dlp/yt-dlp/pull/579#discussion_r680575270